webui/react/src/App.tsx

```diff
@@ -40,6 +41,7 @@ const AppView: React.FC = () => {
   useKeyTracker();
   usePageVisibility();
   useRouteTracker();
+  useErrorBoundary();
```

Comment:
where you thinking of adding a default handler here or leaving that for another pr? if we do add something I guess we can look at the error and do nothing if (isDetError && isHandled)